tests/integration/test_model.py

```diff
@@ -61,6 +70,74 @@ def test_training_and_prediction(c, training_df):
     check_trained_model(c)
 
 
+def test_cuml_training_and_prediction(c, gpu_training_df):
```

Comment:
We should mark these tests for GPU with `@pytest.mark.gpu`

Reply:
Thanks.  Let me mark those. 

Any place i can read up on how `@pytest.mark.gpu` works ?

Reply:
Sure! This page generally goes over how classes/modules can be marked, and how to select/deselect them in pytest runs:

https://docs.pytest.org/en/6.2.x/example/markers.html

One thing to note is that dask-sql skips GPU tests by default unless `--rungpu` is specified:

https://github.com/dask-contrib/dask-sql/blob/96524ee41fd004a92330c32c3be35b5201e6b1c2/conftest.py#L10-L12

That shouldn't conflict with anything changes you've made here, but you should be safe to remove the `pytest.importorskip` lines in individual tests and replace them with a single `try..except ImportError` at the top of the file.

Reply:
Resolved with commithttps://github.com/dask-contrib/dask-sql/pull/330/commits/f583c14e07027599b91a7fcc7e9d6d705e9853ab